gdpr/vendorlist-fetching_test.go

```diff
@@ -139,6 +141,78 @@ func TestVendorListMaker(t *testing.T) {
 	assertStringsEqual(t, "https://vendorlist.consensu.org/v2/archives/vendor-list-v7.json", vendorListURLMaker(7, 2))
 }
 
+func TestDefaultVendorList(t *testing.T) {
```

Comment:
This looks like this test covers two cases. One where the GVL call succeeds and one where it fails. Can you please split them apart for clarity? I know this will duplicate some of the setup, but I think the added clarity is worth it.

Reply:
I will split it into  two